src/components/header/template.njk

```diff
@@ -1,110 +1,120 @@
+              <li class="govuk-header__navigation-item{{ ' govuk-header__navigation-item--active' if item.active }}">
+                <a class="govuk-header__link" href="{{ item.href }}"{% for attribute, value in item.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+                  {{ item.html | safe if item.html else item.text }}
+                </a>
+              </li>
+            {% endif %}
+          {% endfor %}
+        </ul>
+      </nav>
+      {% endif %}
 
-      {% if params.signOutHref %}
-        <nav class="hmrc-sign-out-nav">
-          <a class="govuk-link hmrc-sign-out-nav__link" href="{{ params.signOutHref }}">
-          {% if language == 'cy' %}Allgofnodi{% else %}Sign out{% endif %}
-          </a>
-        </nav>
-      {% elseif params.languageToggle %}
-        <nav class="hmrc-language-select" aria-label="Choose a language">
-          <ul class="govuk-list hmrc-language-select__items">
-            <li class="govuk-header__navigation-item">
-              <a{% if language == 'en' %} aria-current="page"{% else %} class="govuk-header__link"{% endif %}
-                lang="en" hreflang="en" href="{{ params.languageToggle.en.href }}">English</a>
-            </li>
-            <li class="govuk-header__navigation-item">
-              <a {% if language == 'cy' %} aria-current="page"{% else %} class="govuk-header__link"{% endif %}
-                lang="cy" hreflang="cy" href="{{ params.languageToggle.cy.href }}">Cymraeg</a>
-            </li>
-          </ul>
-        </nav>
+        {% if params.signOutHref %}
+          <nav class="hmrc-sign-out-nav">
+            <a class="govuk-link hmrc-sign-out-nav__link" href="{{ params.signOutHref }}">
+            {% if language == 'cy' %}Allgofnodi{% else %}Sign out{% endif %}
+            </a>
+          </nav>
+        {% elseif params.languageToggle %}
+          <nav class="hmrc-language-select" aria-label="Choose a language">
+            <ul class="govuk-list hmrc-language-select__items">
+              <li class="govuk-header__navigation-item">
+                <a{% if language == 'en' %} aria-current="page"{% else %} class="govuk-header__link"{% endif %}
+                  lang="en" hreflang="en" href="{{ params.languageToggle.en.href }}">English</a>
+              </li>
+              <li class="govuk-header__navigation-item">
+                <a {% if language == 'cy' %} aria-current="page"{% else %} class="govuk-header__link"{% endif %}
+                  lang="cy" hreflang="cy" href="{{ params.languageToggle.cy.href }}">Cymraeg</a>
+              </li>
+            </ul>
+          </nav>
+        {% endif %}
+      </div>
       {% endif %}
     </div>
-    {% endif %}
   </div>
+  {% if params.displayHmrcBanner %}
+    <div class="govuk-width-container">
+      <div class="hmrc-banner">
+        <div class="hmrc-organisation-logo">
+          <p class="govuk-body-s">{% if params.language === 'cy' %}
```

Comment:
You've used `language` and `==` elsewhere rather than `params.language` and `===`. Should be consistent.

Reply:
Good spot, this logic was copied in from hmrcBanner, will amend.